siliconcompiler/tools/openroad/openroad.py

```diff
@@ -99,6 +103,20 @@ def setup(chip, mode='batch'):
         chip.add('tool', tool, 'output', step, index, design + '.def')
         chip.add('tool', tool, 'output', step, index, design + '.odb')
         chip.add('tool', tool, 'output', step, index, design + '.lef')
+    elif is_show_screenshot:
```

Comment:
Not looking forward to merging this with my current PR draft:-)

Reply:
If the merge is too onerous, I can take a stab at it, just let me know